<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAddSwitchDialog</class>
 <widget class="QDialog" name="NewAddSwitchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QDialog#NewAddSwitchDialog
{
	border-image:none;
	border-image: url(:/images/images/background_2.jpg);
}

QWidget#widget
{
	border-image: url(:/images/images/whole_back_2.png);
}

QLabel
{
	color: rgb(255, 255, 255);
}

QPushButton
{
	color: rgb(255, 255, 255);
	border-radius: 8px;  
	border-image: url(:/images/images/button_search.png);
}

QLineEdit
{
	color: rgb(0, 0, 0);
}


</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="addressLabel">
          <property name="text">
           <string>带内IP地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="ipAddressLineEdit" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="ip_part1_In"/>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string> .</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ip_part2_In"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string> .</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ip_part3_In"/>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string> .</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ip_part4_In"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="address2Label">
          <property name="text">
           <string>带外IP地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="ipAddress2LineEdit" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="ip_part1_Out"/>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string> .</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ip_part2_Out"/>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string> .</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ip_part3_Out"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string> .</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ip_part4_Out"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>交换机标识</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="switchNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>交换机名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="descriptionLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="locationLabel">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="locationEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="remarkLabel">
          <property name="text">
           <string>端口数量</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="remarkEdit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>28</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>52</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>159</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
